packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -160,7 +178,7 @@ export class VaTextInput {
         {this.label && (
           <label htmlFor="inputField" part="label">
             {this.label}{' '}
-            {this.required && <span class="required">(*Required)</span>}
+            {this.required && <span class="required">{i18next.t('required')}</span>}
```

Comment:
The translation only has the text, not the parentheses and asterisk.

```suggestion
            {this.required && <span class="required">(*{i18next.t('required')})</span>}
```